aten/src/ATen/ParallelNative.h

```diff
@@ -133,28 +132,27 @@ inline scalar_t parallel_reduce(
       internal::_unset_thread_num();
     };
 
-    std::vector<std::shared_ptr<ivalue::Future>> futures(num_tasks);
```

Comment:
What's going to happen to "ivalue::Future"?

Reply:
we still use it in JIT interpreter for fork/join implementation